drake/systems/plants/RigidBodySystem.cpp

```diff
@@ -367,6 +360,53 @@ RigidBodySpringDamper::RigidBodySpringDamper(RigidBodySystem* sys,
   tree->addFrame(frameB);
 }
 
+RigidBodyAccelerometer::RigidBodyAccelerometer(RigidBodySystem *sys,
+                                               const std::string& name,
+                                               const std::shared_ptr<RigidBodyFrame> frame)
+  : RigidBodySensor(sys,name), frame(frame) {  }
+
+Eigen::VectorXd RigidBodyAccelerometer::output(const double &t,
+                                               const KinematicsCache<double> &rigid_body_state,
+                                               const RigidBodySystem::InputVector<double>& u) const
+{
+  auto q = rigid_body_state.getQ();
+  auto v = rigid_body_state.getV();
+  VectorXd x(q.size() + v.size());
+  x << q, v;
+  auto xdd = sys->dynamics(t, x, u);
+  auto tree = sys->getRigidBodyTree();
+  auto v_dot = xdd.bottomRows(v.size());
+  Vector3d sensor_origin = Vector3d::Zero(); //assumes sensor coincides with the frame's origin;
```

Comment:
These will be used in the FLA simulator which has special requirements for a simulated IMU.  The idea is to give simulated IMU measurements to a software-in-the-loop version of the flight controller firmware which is where our inner loop controller lives.  The IMU will also be passed to Pronto along with simulated Lidar scans for state estimation.